.github/workflows/main.yml

```
@@ -23,6 +23,10 @@ jobs:
     env:
       ACTIONS_ALLOW_UNSECURE_COMMANDS: true # to allow setting version env var
       VERSION: "1.1.${{ github.run_number }}"
+      OCTOPUS_API_KEY: ${{ secrets.DEPLOY_API_KEY  }}
+      OCTOPUS_HOST: ${{ secrets.DEPLOY_URL }}
```

It looks like `push-package-action` and `create-release-action` have been brought in line to use `OCTOPUS_URL`.
```suggestion
      OCTOPUS_URL: ${{ secrets.DEPLOY_URL }}
```